cmd/nerdctl/ipfs_compose_linux_test.go

```diff
@@ -111,6 +111,7 @@ volumes:
 }
 
 func TestIPFSComposeUpBuild(t *testing.T) {
+	t.Parallel()
```

Comment:
Is it possible to make a few changes:
* In defer base.Cmd("builder", "prune").Run() --> AssertOk() and wherever we have Run() -> AssertOk() in this function please?
* In the service name in the yaml, is it possible to assign a new name to the service that is fairly unique and captures the context of the test name?
* Is it also possible to listen to a different port as other ports are listening to port 8080 and can lead to flakiness.